Project/Tesseract.vcxproj

```diff
@@ -1,612 +1,612 @@
+    <ClInclude Include="Libs\MathGeoLib\MathGeoLib.h" />
+    <ClInclude Include="Libs\MathGeoLib\MathGeoLibFwd.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\assume.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\BitFuncs.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\BitOps.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\Callstack.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\FixedPoint.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float2.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float3.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float3x3.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float3x4.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float4.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float4d.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float4x4.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float4x4_neon.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float4x4_sse.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float4_neon.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\float4_sse.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\FloatCmp.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\grisu3.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\InclWindows.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\Interpolate.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MathAll.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MathConstants.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MathFunc.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MathLog.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MathNamespace.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MathTypes.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MatrixProxy.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MiniFloat.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\MultiLineMacro.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\myassert.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\Polynomial.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\Quat.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\quat_simd.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\Rect.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\Reinterpret.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\simd.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\SSEMath.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\sse_mathfun.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\Swap.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\TransformOps.h" />
+    <ClInclude Include="Libs\MathGeoLib\Math\vec2d.h" />
+    <ClInclude Include="Libs\MathGeoLib\Time\Clock.h" />
+    <ClInclude Include="Source\Utils\PoolMap.h" />
+    <ClInclude Include="Source\Utils\StringBlocksParser.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="Libs\imgui\imgui_tables.cpp" />
+    <ClCompile Include="Source\Components\ComponentParticleSystem.cpp" />
```

Comment:
PanelResource has been removed from ClCompiles. It should be readded:
`<ClCompile Include="Source\Panels\PanelResource.cpp" />`